### ISSUE TITLE
how to plot graph for different bitrates vmaf score, that is rate distortion graph?

### ISSUE THREAD

**Issue by vainaijr** [Netflix/vmaf#347]
the only way I know is through matplotlib, running experiment on multiple bitrates, generating multiple output files, and then plot vmaf vs bitrate using matplotlib.

**Comment by slhck** [comment]
I don't believe this is within the scope of this project. You can use any tool of your choice to create such graphs. The way you've described seems like what most people would do.
